modules/node-shared/src/main/scala/org/tessellation/node/shared/cli/CliMethod.scala

```diff
@@ -61,6 +61,10 @@ object CliMethod {
 
   val forkInfoStorageConfig: ForkInfoStorageConfig = ForkInfoStorageConfig(10)
 
+  val doubleSignDetectConfig: DoubleSignDetectConfig = DoubleSignDetectConfig(
+    maxDetectionDelta = 10L
```

Comment:
@ryle-goehausen , please verify that this is the desired value for the double signing-rollback cutoff.

Reply:
Yes that's fine for now, we can adjust after viewing metrics.

Reply:
@ryle-goehausen I think you should run the cluster and observe the metrics, then update the PR with the correct value. Otherwise, the change with incorrect values may enter mainnet after merging this PR.